src/core/server/saved_objects/migrationsv2/migrations_state_action_machine.ts

```diff
@@ -196,16 +197,19 @@ export async function migrationStateActionMachine({
   } catch (e) {
     await cleanup(client, executionLog, lastState);
     if (e instanceof EsErrors.ResponseError) {
-      logger.error(
-        logMessagePrefix + `[${e.body?.error?.type}]: ${e.body?.error?.reason ?? e.message}`
-      );
+      // Log the failed request. This is very similar to the
+      // elasticsearch-service's debug logs, but we log everything in single
+      // line until we have sub-ms resolution in our cloud logs. Because this
+      // is error level logs, we're also more careful and don't log the request
+      // body since this can very likely have sensitive saved objects.
+      const req = getRequestDebugMeta(e.meta);
+      const failedRequestMessage = `Unexpected Elasticsearch ResponseError: statusCode: ${
+        req.statusCode
+      }, error: ${getErrorMessage(e)}, method: ${req.method}, url: ${req.url}`;
```

Comment:
Shouldn't we change the order to align with the error message in ES client: `statusCode, method, URL, error message` https://github.com/elastic/kibana/blob/2a3113fee4b7b95e1ce053476bb9a530ad4b81b8/src/core/server/elasticsearch/client/configure_client.test.ts#L311-L313 ?

Reply:
yeah good point, I initially included the body and because the body might easily be large enough to cause the log line to truncate I wanted to put the error in the front of the log line. But after I removed the body for security reasons it makes sense to keep the same order.